recipes/lpython/version.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/build0.sh b/build0.sh
+index d1f1c279e..e82bb33d0 100755
+--- a/build0.sh
++++ b/build0.sh
+@@ -3,9 +3,6 @@
+ set -e
+ set -x
+ 
+-# Generate the `version` file
+-ci/version.sh
+-
+ # Generate a Python AST from Python.asdl (Python)
+ python grammar/asdl_py.py
+ # Generate a Python AST from Python.asdl (C++)
+diff --git a/build0_win.xsh b/build0_win.xsh
+index 8530d835b..99ee6e260 100644
+--- a/build0_win.xsh
++++ b/build0_win.xsh
+@@ -1,9 +1,5 @@
+ #!/usr/bin/env xonsh
+ 
+-#bash ci/version.sh
+-version=$(git describe --tags --dirty).strip()[1:]
+-echo @(version) > version
+-
+ # Generate a Python AST from Python.asdl (Python)
+ python grammar/asdl_py.py
+ # Generate a Python AST from Python.asdl (C++)
+diff --git a/version b/version
+new file mode 100644
+index 000000000..54d1a4f2a
+--- /dev/null
++++ b/version
+@@ -0,0 +1 @@
++0.13.0
```

Comment:
This should be generated in build.sh.
```
echo $PKG_VERSION > version
```

Reply:
Fixed in fba9ec5ccc534b346d9ddd173e82f3070a6abc9f.